### DISCUSSION TITLE
Views and projections

### DISCUSSION THREAD

**Discussion by ricardoekm** [visgl/deck.gl#7886] [Q&A]
Hi everyone, 
I'm wrapping my head around views and projection in Deck. Some points are not clear to me:

- Why the Ortographic View Port uses the Web Mercator projection from Math.GL? https://github.com/visgl/deck.gl/blob/818751a9af76b41cfdea020a9dff06bf409c6fac/modules/core/src/viewports/orthographic-viewport.ts#L133
- What makes the coordinates and map repeat themselves as the user pan continuously in the Web Mercator View port and why that doesn't happen in the Ortographic View? The subviewports are disabled by default in the Web Mercator so I couldn't find any relevant difference between the two.
https://github.com/visgl/deck.gl/blob/818751a9af76b41cfdea020a9dff06bf409c6fac/modules/core/src/viewports/web-mercator-viewport.ts#L121

Thanks!

**Comment by Pessimistress** [comment]
`pixelsToWorld` has nothing to do with web mercator projection, it is a utility that converts a point from the screen space to the common space.

Subviewports are generated by https://github.com/visgl/deck.gl/blob/818751a9af76b41cfdea020a9dff06bf409c6fac/modules/core/src/viewports/web-mercator-viewport.ts#L223

**Reply by ricardoekm** [reply]
Thanks for the reply @Pessimistress ! What exactly makes the web-mercator-view port and its coordinates rotate continuously as the use pans? I mean as you keep panning to right you'll see the US multiple times.

**Reply by Pessimistress** [reply]
Every copy of the world (longitude [-180, 180]) maps to a sub viewport.
